<div class="section">

    <section id="content">
        <h1>Cell Line Identification</h1>
        <h2>Methods for Authentication of Human Cell Cultures and Tissues</h2>

<h3>Cancer Center, Texas Tech University Health Sciences Center School of Medicine.</h3>
<h3 style="font-size:150%;"><a href="https://www.txccr.org/submitsample.php"><strong>Now Accepting Samples for DNA authentication!</strong></a></h3>
<br>

<p>

<strong>Submit a Sample for Testing.</strong><br><br>

The TXCCR is now accepting samples for STR cell line and PDX DNA authentication as well as cell line mycoplasma bacteria screenings. Submissions are limited to
material that has been provided either through CCcells.org and/or TXCCR.org. Please visit <a href="http://www.txccr.org/submitsample.php">here </a> to fill out the submission form.
<br><br>
Basic STR DNA authentication is $30.00 per sample and includes:
<br>
STR Testing - DNA extraction, quantification, amplification and fragment separation. STR profile determination (9 STR loci + Amelogenin)
<br>
Data analysis - Comparison of profile to known samples or an archived database entry.
<br><br>
Report - Email notification to contributor of match/no match. Comments regarding allele drop-out, drop-in, and peak imbalance, upon request. Certificate of Analysis that includes STR profile and comparison result upon request.
<br>
Result reporting up to 8 weeks of sample receipt depending on sample backlog. All sample(s) submitted will be consumed during testing.
<br>
Basic Mycoplasma supernatant assay is $15.00 per sample and includes:
<br><br>
Testing - Rapid screening of cell supernatant to detect up to 44 mollicute species of mycoplasma.
<br><br>
Reporting - Email notification of positive or negative result in relation to controls. Intended for general routine testing and no official of certification of analysis will be provided.
<br><br>
Result reporting within 6-8 weeks of sample receipt. All sample(s) submitted will be consumed during testing. <br>
<a href="http://www.txccr.org/submitsample.php">Submit a Sample.</a>
<br><br>
</p>
<h2>Introduction</h2>
<p>Growth of cells in cultures or xenografts provides many opportunities for mislabeling or cross-contamination in spite of best laboratory practice. Contamination can cause mixtures with resulting outgrowth and subsequent replacement by the more aggressive, dominant line. Therefore the genetic identities of cell cultures and xenografts should be validated when initially established and verified periodically to insure the integrity of research conducted with the lines (1). On November 28, 2007, the National Institutes of Health issued a notice that in part made the following statements (2);</p>

<p class="indent4">"We are convinced by the body of evidence developed by Dr. Nardone and his colleagues that misidentification of cell cultures is a serious problem. Grant applications that fail to employ such practices (cell culture authentication to prove true identity of the culture) would not be considered of the highest quality and such manuscripts would not fare well in the journal review process. We encourage all reviewers to consider these issues carefully in order to protect and promote the validity of the science we support."</p>

<p>The Childhood Cancer Repository has long employed methods of molecular identification of cell lines and xenografts. After study of the various alternatives, the laboratory chose to implement the Applied Biosystems AmpF/STR&reg; Identifiler<sup>TM</sup> system for genetic identification of cell lines and xenografts. The Identifiler<sup>TM</sup> kit was developed specifically for use in forensic laboratories throughout the world (3) as a standardized method for assessing short tandem repeat (STR) loci and software to reduce analyzed loci to a numeric "molecular fingerprint". This method is routinely employed in the Childhood Cancer Repository (4).</p>

<p class="ctr"><a href="pix/clid-1-b.jpg" target="_blank"><img src="pix/clid-1-s.jpg" width="350" height="297" alt="" border="0"></a></p>
<p class="indent3"><i>Figure 1. Loci designated for use in the Combined DNA Identification System (CODIS). These loci were selected by the Federal Bureau of Investigation (FBI) after years of research and collaboration with other laboratories. The CODIS database is maintained by the FBI and contains offender DNA profiles generated by Federal, State and local crime laboratories. Comparing these 13 required forensic loci to those used in the Identifiler<sup>TM</sup> kit you will see that all loci from the FBI CODIS are included in the Identifiler<sup>TM</sup> kit, and Identifiler<sup>TM</sup> employs 2 additional loci (D2S1338 and D19S433) (Figure 2).</i></p>

<p>The Identifiler<sup>TM</sup> kit is a human specific, multiplex PCR-based system testing 15 different short tandem repeat (STR) polymorphic loci.  A 16<sup>th</sup> locus, intron 1 of the amelogenin gene, which is located on the X and Y chromosomes, provides gender identification.  A 6 base pair deletion on the X chromosome allele of amelogenin allows electrophoretic distinction between DNA from X or Y chromosomes.  Since females are X,X, only a single peak is observed when testing female DNA whereas males, which possess both X and Y chromosomes, exhibit two peaks with a standard amelogenin test (3).</p>

<p>The loci chosen have high heterozygosity, regular repeat units, distinguishable alleles and robust amplification (2). The primers used during PCR have fluorescent dyes attached and the resulting amplicons are laser-detected as fragments of different lengths during capillary electrophoresis. Then, proprietary software is used to establish profiles based upon the number of repeated sequences at each allele of each locus. The loci used are located on independently assorting chromosomes and the probability of this particular assay for identifying tissues from two different individuals as identical averages from 1 in 7.64 x 10<sup>17</sup> to 1 in 2.76 x 10<sup>16</sup>, depending on the race (5).</p>

<p class="ctr"><a href="pix/clid-1a-b.jpg" target="_blank"><img src="pix/clid-1a-s.jpg" width="350" height="191" alt="" border="0"></a></p>
<p class="indent3">Figure 1A. Example of short tandem repeat (STR) sequence of DNA (TCAT). This allele would type as a "6".</p>

<h2>Materials and methods</h2>

<p>DNA samples and STR profiling</p> 

<p>Human cell culture material, tumors and whole blood samples are the most common samples submitted to the COG Core Laboratory for STR typing. Presumed xenograft tumors from mice are also submitted to determine if the tissue is of human or mouse origin since this assay is human specific. Cell culture material is submitted as dry pellets consisting of 1 x 10<sup>6</sup> to 5 x 10<sup>6</sup> cells and placed in the Core Laboratory -20&deg;C freezer. Tumor samples are submitted as approximately 25 mg of tissue pieces in microcentrifuge tubes and whole blood samples are submitted as approximately 1 ml in microcentrifuge tubes. The contributor logs the samples into the STR Samples Request book at the time of the submission.</p>

<p>DNA is isolated from the cell pellets or tissues with the Qiagen&reg; DNeasy&reg; kit and quantitation of the extracts is performed using a NanoDrop<sup>TM</sup> spectrophotometer (Thermo Fisher Scientific Inc). DNA amplification and fragment fluorescent labeling is performed using the GeneAmp&reg; PCR System 9700 thermal cycling instrument and the Identifiler<sup>TM</sup> AmpF/STR&reg; PCR kit. Approximately 0.125 ng/&micro;l of purified DNA is used for amplification (6).</p>

<p class="ctr"><a href="pix/clid-2-b.jpg" target="_blank"><img src="pix/clid-2-s.jpg" width="350" height="253" alt="" border="0"></a></p>
<p class="indent3"><i>Figure 2. Loci amplified when using the Identifiler<sup>TM</sup> kit.</i></p>

<p>Capillary electrophoresis and fragment detection is performed with a 3130xl Genetic Analyzer with Data Collection software v 3.0 (Applied Biosystems). GeneMapper ID&reg; Version 3.2.1 (Applied Biosystems) software is used to assign fragment repeat numbers which becomes collectively the STR type for that individual (6).</p>

<p class="ctr"><a href="pix/clid-3-b.jpg" target="_blank"><img src="pix/clid-3-s.jpg" width="380" height="258" alt="" border="0"></a></p>
<p class="indent3"><i>Figure 3. Thermal cycler and ABI 3130xl Genetic Analyzer, 16 capillaries.</i></p>

<p class="ctr"><a href="pix/clid-4-b.jpg" target="_blank"><img src="pix/clid-4-s.jpg" width="350" height="363" alt="" border="0"></a></p>
<p class="indent3"><i>Figure 4. Example of GeneMapper ID&reg; STR data.</i></p>

<h2>Database Searches</h2>

<p>The result of DNA STR typing done for the COG Repository and TTUHSC includes the complete 16 loci set of the sample, which are entered into an MySQL database
     and the results are searchable on the <a href="https://strdb.cogcell.org/" target="_blank">strdb.cogcell.org</a> website. This database currently contains ~ 
     2500 previous STR profiles from cell cultures and tissues from the COG Repository, cell lines established at TTUHSC and UT Southwestern Medical Center, and
      STR data from other repositories (<a href="http://www.atcc.org/" target="_blank">ATCC</a>, <a href="http://www.dsmz.de/" target="_blank">DSMZ</a>, 
      <a href="http://cellbank.nibio.go.jp/" target="_blank">JCRB</a>, and <a href="http://www.riken.go.jp/engn/" target="_blank">Riken</a>). Some of the data 
      employ 8 loci profiles. The sample contributor is advised of any matches by email. This database is employed internally by the
       <a href="https://www.cccells.org/" target="_blank=">Childhood Cancer Repository</a>, <span style="font-size: 85%">powered by Alex's Lemonade Stand 
        Foundation</span> and the <a href="https://www.txccr.org/" target="_blank">TXCCR</a> when establishing new lines or validating established cultures.
         The database is available online (<a href="https://strdb.cogcell.org/" target="_blank">https://strdb.cccells.org/</a>) for any investigator to use to
          identify a cell line or verify that a new line is unique with respect to the database of known lines.</p> 

<p class="ctr"><a href="pix/clid-5-b.jpg" target="_blank"><img src="pix/clid-5-s.jpg" width="350" height="244" alt="" border="0"></a></p>
<p class="indent3"><i>Figure 5. Example of matches obtained when a new sample profile was entered into the database with ten loci shown. The 2008 cell line is the same as the cell line first typed in 2005. The "p" in the cell line name stands for the passage number.</i></p>

<h2>Acknowledgements</h2>

<p>This web page was based on a poster designed by Larry Starr, TTUHSC, Medical Photography, Lubbock, TX and Charles Linch, TTUHSC Cancer Center.</p>

<h2>References</h2>

<ol>
<li>Newsfocus: Cases of mistaken identity. Science Vol 315, page 928-931, Feb 16, 2007 </li>
<li><a href="http://grants.nih.gov/grants/guide/notice-files/NOT-OD-08-017.html" target="_blank">http://grants.nih.gov/grants/guide/notice-files/NOT-OD-08-017.html</a></li>
<li><a href="http://www.cstl.nist.gov/strbase/" target="_blank">www.cstl.nist.gov/strbase/</a></li>
<li>Cabral DJ, Feaman HV, Sheard MA, Reynolds CP: Short tandem repeat genotyping using Identifiler<sup>TM</sup> is a reproducible and cost-effective method for establishing and monitoring identity of pediatric cancer cell lines. Proc Amer Assoc Cancer Res 48:96, 2007.</li>
<li><a href="http://www3.appliedbiosystems.com/cms/groups/applied_markets_support/documents/generaldocuments/cms_041201.pdf" target="_blank">http://www3.appliedbiosystems.com/cms/groups/.../cms_041201.pdf</a></li>
<!-- <li><a href="dl/prot/STR-SOP_cpr_edits.pdf">Standard Operating Procedure, Procedure for Establishing Unique Genetic Identity of Cell Lines or Xenografts</a>, Cancer Center Cell Culture and Xenograft Core, TTUHSC, Dept. of Cell Biology and Biochemistry, 2008</li> -->
<li><a href="dl/prot/2015-11-18/SOP_25_Short_Tandem_Repeat_STR_Profiling_GP10.pdf" target="_blank">SOP 25 - Short Tandem Repeat STR Profiling GP10</a></li>
</ol>
                           
</section><!-- #end content area -->


   